Zend/zend_API.c

```diff
@@ -3324,7 +3324,8 @@ ZEND_API zend_bool zend_is_callable_ex(zval *callable, zend_object *object, uint
 						((fcc->function_handler->common.fn_flags & ZEND_ACC_CALL_VIA_TRAMPOLINE) ||
 					     fcc->function_handler->type == ZEND_OVERLOADED_FUNCTION_TEMPORARY ||
 					     fcc->function_handler->type == ZEND_OVERLOADED_FUNCTION)) {
-						if (fcc->function_handler->type != ZEND_OVERLOADED_FUNCTION) {
+						if (fcc->function_handler->type != ZEND_OVERLOADED_FUNCTION && 
+							fcc->function_handler->common.function_name) {
```

Comment:
Similar checks are repeated in a few more places. Maybe they also need to be changed?

Reply:
@nikic sounds reasonable, thanks. Just updated it.